.github/workflows/deploy.backend-api.yml

```diff
@@ -0,0 +1,56 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
+# More GitHub Actions for Azure: https://github.com/Azure/actions
+
+name: Build and Deploy Backend API
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  AZURE_WEBAPP_PACKAGE_PATH: './app/Monitor'
+  DOTNET_VERSION: '8.x'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: ${{ env.DOTNET_VERSION }}
+      
+      - name: dotnet build and publish
+        run: |
+          dotnet restore
+          dotnet build --configuration Release
+          dotnet publish -c Release --property:PublishDir='${{ env.AZURE_WEBAPP_PACKAGE_PATH }}'
+      
+      - name: Upload artifact for deployment job
+        uses: actions/upload-artifact@v3
+        with:
+          name: .net-app
+          path: ${{ env.AZURE_WEBAPP_PACKAGE_PATH }}
+  
+  # Deploy UAT
+  deploy-uat:
+    runs-on: ubuntu-latest
+    needs: build
+    environment: uat
+
+    steps:
```

Comment:
You're missing the db migrations steps here - see EquineCastration: https://github.com/EquineCastration/Equine-Castration/blob/main/.github/workflows/deploy.dev.equine-castration.yml

Reply:
Also it's definitely worth trying to get closer to the structure in that repo as well. 
With build/publish/deploy being singularly responsible for their role in the workflow. And most importantly not having multiple builds of the same apps in the deploy workflow.